api/bazel/api_build_system.bzl

```diff
@@ -125,6 +127,11 @@ def api_proto_library(name, visibility = ["//visibility:private"], srcs = [], de
         deps = [_LibrarySuffix(d, _CC_SUFFIX) for d in deps],
         external_deps = [
             "@com_google_protobuf//:cc_wkt_protos",
+            # TODO(dio): Running `bazel build @envoy_api//envoy/...` on mac
```

Comment:
Is this similar issue to https://github.com/bazelbuild/bazel/issues/5163 ?

Though I don't think we should add this here to every `api_proto_library`.

Reply:
OK, I will remove the comment.